parsons/community/community.py

```diff
@@ -0,0 +1,101 @@
+import logging
+from parsons.utilities.api_connector import APIConnector
+from parsons.utilities import check_env
+from parsons.etl import Table
+
+logger = logging.getLogger(__name__)
+
+COMMUNITY_API_ENDPOINT = "https://dl.community.com/download/v1/files/"
+
+
+class Community(object):
+    """
+    Instantiate class.
+
+       `Args:`
+            community_client_id: str
+                The Community provided Client ID. Not required if ``COMMUNITY_CLIENT_ID`` env
+                variable set.
+            community_access_token: str
+                The Community provided access token. Not required if ``COMMUNITY_ACCESS_TOKEN`` env
+                variable set.
+            community_uri: str
+                The URI to access the  API. Not required, default is
+                https://dl.community.com/download/v1/files/. You can set an ``COMMUNITY_URL`` env
```

Comment:
Sorry I didn't catch this before. You can format external links using [this syntax](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages).